pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -132,3 +145,29 @@ func (oe openshiftExtension) Finalize(ctx context.Context, comp v1alpha1.TektonC
 func configOwnerRef(tc v1alpha1.TektonInstallerSet) metav1.OwnerReference {
 	return *metav1.NewControllerRef(&tc, tc.GetGroupVersionKind())
 }
+
+func removeOwnerRefFromPreExistingSA(ctx context.Context, kc kubernetes.Interface) error {
+	allSAs, err := kc.CoreV1().ServiceAccounts("").List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, sa := range allSAs.Items {
+		if sa.Name == "pipeline" && !nsRegex.MatchString(sa.Namespace) {
+			sa.SetOwnerReferences([]metav1.OwnerReference{})
```

Comment:
should we replace it with TektonConfig ref 🤔 
so they will be deleted when TC is deleted?
WDYT

Reply:
This makes sense to me, its anyway created by operator and should be deleted with it.

Reply:
PR Updated..